pyproject.toml

```diff
@@ -21,11 +21,13 @@ dependencies = [
     "numpy >=1.25",
     "photutils >=2.1.0",
     "requests >=2.26",
-    # "roman_datamodels>=0.26.0,<0.27.0",
-    "roman_datamodels @ git+https://github.com/spacetelescope/roman_datamodels.git",
+    # "roman_datamodels>=0.23.1,<0.24.0",
+    "roman_datamodels @ git+https://github.com/mairanteodoro/roman_datamodels.git@mteodoro_sky_var_rdnoise",
```

Comment:
Are there datamodel changes needed? If so would you link the PR?

Reply:
Jinx! https://github.com/spacetelescope/romancal/pull/1691/commits/9c172e915abc2ae4304da1c90b66106a1a4b620e